README.md

```
@@ -311,6 +311,274 @@ print(json_str)
+# ✅ Metadata is instantly available!
+```
+
+---
+
+### 🗂 Example Config File  
+
+```plaintext
+!CONFIG FILE!
+
+Config_User|dict:
+	typechecking|dict:
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)